packages/x-components/src/x-modules/queries-preview/components/__tests__/query-preview.spec.ts

```diff
@@ -101,8 +103,23 @@ describe('query preview', () => {
     expect(getXComponentXModuleName(wrapper.vm)).toBe('queriesPreview');
   });
 
+  it('no sends the `QueryPreviewRequestUpdated` event', () => {
```

Comment:
Hi! I think it would be nice to test that `load` is being emitted with the `queryPreviewInfo.query`. I would update the test definition this way, to point to the prop & the alternative behavior that provoques:
```suggestion
  it('no sends the `QueryPreviewRequestUpdated` event if saveCache is true, but emits load', () => {
```